core/src/main/res/values/strings.xml

```diff
@@ -443,17 +443,13 @@
     <string name="pref_theme_title_dark">Dark</string>
     <string name="pref_theme_title_trueblack">Black (AMOLED ready)</string>
     <string name="pref_episode_cache_unlimited">Unlimited</string>
-    <string name="pref_gpodnet_authenticate_title">Login</string>
-    <string name="pref_gpodnet_authenticate_sum">Login with your gpodder.net account in order to sync your subscriptions.</string>
-    <string name="pref_gpodnet_logout_title">Logout</string>
     <string name="pref_gpodnet_logout_toast">Logout was successful</string>
     <string name="pref_gpodnet_setlogin_information_title">Change login information</string>
     <string name="pref_gpodnet_setlogin_information_sum">Change the login information for your gpodder.net account.</string>
     <string name="pref_gpodnet_sync_changes_title">Synchronize now</string>
-    <string name="pref_gpodnet_sync_changes_sum">Sync subscription and episode state changes with gpodder.net.</string>
     <string name="pref_gpodnet_full_sync_title">Force full synchronization</string>
-    <string name="pref_gpodnet_full_sync_sum">Sync all subscriptions and episode states with gpodder.net.</string>
-    <string name="pref_gpodnet_login_status"><![CDATA[Logged in as <i>%1$s</i> with device <i>%2$s</i>]]></string>
+    <string name="pref_nextcloud_gpodder_login_status"><![CDATA[Logged in as <i>%1$s</i>. <br/><br/>%2$s]]></string>
```

Comment:
Do we still need the italic font? I think I would just use quotation marks to not confuse translators. Couldn't the second string that is appended to this be specified directly here?

Reply:
> Do we still need the italic font? I think I would just use quotation marks to not confuse translators.

I would prioritise whatever is seen by end-users over ease for translators. I imagine italics tags are fine (given that they also have to deal with CDATA stuff, newline tags & variables.